### PR TITLE
[PARTIAL REVERT] Removes round start lights off (keeps empty departments)

### PR DESCRIPTION
**Pull request by Moltijoe**
partially reverts #20493

# Why is this good for the game?
While initially cool, it has caused issues for mappers and now with the new lighting rework pr it puts a huge strain on the server

# Testing
round just started and the lights are on except for the empty departments
image
image

:cl:  
rscdel: Lights no longer start off except for empty departments
tweak: adds a cooldown to light flickering from explosions/etc
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
while you're at it you might want to make the lights flickering from explosions thing have a bit of a cooldown so meteor swarms dont melt the server by turning everything on and off every second or two

**Comment by Moltijoe** [comment]
> while you're at it you might want to make the lights flickering from explosions thing have a bit of a cooldown so meteor swarms dont melt the server by turning everything on and off every second or two

done

**Comment by Aquizit** [comment]
Could you make it so as long as there is one head of staff at round start that the bridge stays on? I think it currently might only do it for the captain -- or no one since no one starts on the bridge

**Comment by ShoziOffical** [comment]
> Could you make it so as long as there is one head of staff at round start that the bridge stays on? I think it currently might only do it for the captain -- or no one since no one starts on the bridge

Yeah i have thought about that long time ago

**Comment by Moltijoe** [comment]
> Yeah i have thought about that long time ago

other heads are supposed to light up bridge, but there seems to be a bug that i have no clue what the issue is

**Review by muon36 — APPROVED** [review]
Looks good

**Comment by ShoziOffical** [comment]
> Looks good

Imposter sus as hell

**Review by SapphicOverload — APPROVED** [review]
Code looks fine, still waiting on HD approval though

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**
